chef/data_bags/crowbar/bc-template-nova_dashboard.json

```diff
@@ -32,7 +32,7 @@
       },
       "can_set_mount_point": false,
       "can_set_password": false,
-      "use_keystone_v3": false,
+      "multi_domain_support": true,
```

Comment:
As long as we don't have keystone V3 as default, `multi_domain_support` should be `false`